descartes_planner/src/dense_planner.cpp

```diff
@@ -6,6 +6,7 @@
  */
 
 #include <descartes_planner/dense_planner.h>
+#include <boost/make_shared.hpp>
```

Comment:
Yes, but `std` and `boost` shared pointers do not guarantee compatibility. We use `boost` pointers all over the base classes, so we'd need to do a big find-and-replace operation. Probably best for another PR.